pages/common/encfs.md

```diff
@@ -0,0 +1,24 @@
+# encfs
+
+> Encrypted virtual filesystem.
+> More information: <https://github.com/vgough/encfs>.
+
+- Initialize or mount an encrypted filesystem:
+
+`encfs {{/path/to/cipher_dir}} {{/path/to/mount_point}}`
+
+- Initialize an encrypted filesystem with standard settings:
+
+`encfs --standard {{/path/to/cipher_dir}} {{/path/to/mount_point}}`
+
+- Run encfs in foreground instead of spawning a daemon:
+
+`encfs -f {{/path/to/cipher_dir}} {{/path/to/mount_point}}`
```

Comment:
```suggestion
`encfs -f {{path/to/cipher_dir}} {{path/to/mount_point}}`
```

Reply:
@michaeldel you only removed 1 slash `/`.

Reply:
@navarroaxel see https://github.com/tldr-pages/tldr/pull/5513#discussion_r602779150

Reply:
@navarroaxel see https://github.com/tldr-pages/tldr/pull/5513#discussion_r602778748